### DISCUSSION TITLE
Practicas a realizar

### DISCUSSION THREAD

**Discussion by CharlieCCR** [Launch-X-Latam/MisionFrontEnd#192] [Q&A] [answered]
¡Hola a todos!
Hace menos de una semana me incorpore tanto al teams y al GitHub, me he estado poniendo al dia viendo las grabaciones que ya se han hecho y por supuesto, revisar todo el material del Onboarding pero, me gustaría preguntar ¿Hasta este momento, ya nos han pedido realizar practicas, como diseñar nuestra propia aplicación o programar algo? Lo pregunto ya que he visto a varios de ustedes subiendo sus practicas pero no se si eso lo están haciendo cada quien de manera autodidacta o si en alguna de las charlas en vivo el MC nos a pedido o motivado a realizar alguna practica en especifico.

Me parece que este tipo de pregunta ya se hizo pero igualmente pregunto por si hay alguna pregunta que no se haya tocado aun y pueda ayudar a alguien en mi misma situación.

**Comment by Evelo00** [comment, marked as answer]
Hola! buen dia.
Si te fijas en la parte de [INTRO](https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/01%20-%20INTRO)  hay una parte que dice "Practicas", Ahi encontraras el proceso de realizacion (Sin codigo, solo la construccion de la aplicacion) De Abogabot, que es el ejemplo para este curso, pero si tienes un proyecto personal y quieres llevarlo a cabo siguiendo los puntos de las practicas eres libre de hacerlo. subelo a tu repositorio de github al terminar cada punto, algun momento lo pediran :) Suerte!
